src/test/java/com/SAS/facility/FacilityIT.java

```diff
@@ -0,0 +1,73 @@
+package com.SAS.facility;
+
+import com.SAS.game.Game;
+import com.SAS.team.Team;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FacilityIT {
+
+    private Facility facility;
+    private Game game;
+    private Team team;
+
+    @BeforeEach
+    void setUp() {
+        facility = new Facility();
+        game = new Game();
+        team = new Team();
+    }
+
+    @Test
+    void addValidGameToFacilityBooleanCheck() {
+        facility.addGame(game);
+        boolean result = facility.addGame(game);
+        assertTrue(result);
+    }
+
+    @Test
+    void addValidGameToFacilityListCheck() {
+        facility.addGame(game);
+        int result = facility.getGamesList().size();
+        assertEquals(1, result);
+    }
+
+    @Test
+    void addInvalidGameToFacilityBooleanCheck() {
+        game = null;
+        facility.addGame(game);
```

Comment:
I think it's better to send null in the brackets and not turn game into null since someone can choose to run the tests in different order and then you first tests will fail

Reply:
same as here. the objects get initialized before every test.